docs/docs/snapshots/azure.md

```diff
@@ -91,4 +91,27 @@ Remember to substitute all values in this file that looks like a `<tag>` such as
 | subscription-id | Id of the subscription to inspect, must be the same as the user described in the **Azure** connector file |
 | master-Snapshot-Id | Name of the master snapshot id, you will use this in master test files. This name should be unique in the container |
 | resource-provider | the type of the resource we want to capture during the crawl process. In the `azure` connector, this should be the resource type supported by Azure. |
-| collection-name | in which collection in database we want to store this resource type. Usually it is a good practice to have a separate collection for each type. But based on the scale of implementation, you may want to put multiple resource types in a single collection |
\ No newline at end of file
+| collection-name | in which collection in database we want to store this resource type. Usually it is a good practice to have a separate collection for each type. But based on the scale of implementation, you may want to put multiple resource types in a single collection |
+
+# Master Snapshot Configuration File in remote git
+It is possible to centrally manage your master snapshot configuration files from a git repository. When you are doing that, you can make sure one version of truth is circulating in your different collections and you can manage it centrally.
+
+The format for the remote git master snapshot configuration file is like this:
+
+```
+    {
+    "$schema": "",
+    "contentVersion": "1.0.0.0",
+    "fileType": "masterSnapshot",
+    "connector": "prancer_compliance_structure",
+    "remoteFile": "azure/iac/master-config.json",
+    "connectorUsers": [
+        {
+        "id": "USER_1",
```

Comment:
We have to add more information about what is `USER_1` and need to add sample json of git master snapshot file.

Reply:
Added USER_1 description.